packages/lib/sdk/src/configuration/getEvidenceConfig.legacy.js

```diff
@@ -11,7 +11,8 @@ export const getEvidenceConfigLegacy = () => {
 		const filepath = path.join(projectRoot, 'evidence.plugins.yaml');
 		fileContents = fs.readFileSync(filepath, 'utf-8');
 	} catch (e) {
-		throw new EvidenceError('Could not find an evidence.plugins.yaml file.');
+		console.error('Could not find an evidence.plugins.yaml file.');
```

Comment:
Any specific reason this doesn't throw anymore? We moved over from having a default to throwing when migrating to the sdk

Reply:
Reverted [b1eb047](https://github.com/evidence-dev/evidence/pull/2628/commits/b1eb0474c507215036347d8d03c9cf3e5bbd9f5f)